sdk/redis/Microsoft.Azure.Management.Redis/tests/Utility.cs

```diff
@@ -35,6 +35,12 @@ public static RedisManagementClient GetRedisManagementClient(string responseStri
             return client;
         }
 
+        public static bool IsLiveTest()
```

Comment:
good to have this function in test common

Reply:
@ArthurMa1978 I couldn't understand clearly what you mean by test common.
This Utility.cs is something we use as a common util for our solution.
Can you point me to the right file?